### PR TITLE
Remove format and format2.

### PR DESCRIPTION
**Pull request by ndmckinley**
This should produce some formatting changes in Ansible.

-----------------------------------------------------------------
# [all]
## [ansible]
Formatting changes related to upstream code generator cleanup.

### CODE REVIEW

**Comment by ndmckinley** [comment]
This is related to https://github.com/GoogleCloudPlatform/magic-modules/issues/1181, https://github.com/GoogleCloudPlatform/magic-modules/issues/1179, and https://github.com/GoogleCloudPlatform/magic-modules/issues/1159.

**Comment by ndmckinley** [comment]
https://github.com/modular-magician/ansible/pull/162#issuecomment-453708367 is relevant to this issue also!

File: provider/ansible.rb [hunk 1]
```diff
@@ -269,11 +271,13 @@ def generate_resource_tests(data)
...
         name = module_name(data[:object])
+        path = File.join(target_folder,
+                         "test/integration/targets/#{name}/tasks/main.yml")
         generate_resource_file data.clone.merge(
           default_template: 'templates/ansible/integration_test.erb',
-          out_file: File.join(target_folder,
-                              "test/integration/targets/#{name}/tasks/main.yml")
+          out_file: path
         )
+        %x(python3 -m black --line-length 160 -S #{path} 2> /dev/null) if path.end_with?('.py')
```
Comment (rambleraptor) [inline provider/ansible.rb]: Can we have some kind of `run_formatter` function that does the following:

1) Runs the command.
2) If command fails with error, prints error and keeps going.
3) If command doesn't exist, specifically says command doesn't exist and keeps going

Basically, I don't want to have MM users have to install gofmt, black, and rubocop. MM will split out less pretty-code, but it won't crash.

Comment (ndmckinley) [reply provider/ansible.rb]: Done.  The output code likely won't compile in terraform and likely won't produce valid output in python, but it'll come out anyhow.

File: provider/ansible/request.rb [hunk 2]
```diff
@@ -72,38 +72,38 @@ def properties_with_classes(properties)
...
-            ],
-            [
-              "#{unicode_string(prop.api_name)}:",
-              indent(request_output(prop, hash_name, module_name).to_s, 4)
-            ]
-          ], 0, indent, 160
-        )
+      def request_property(prop, hash_name, module_name, indent_width)
+        # We are still, somewhat in the code-formatting business.  This code is
+        # here because autopep8 cannot handle one particular case: a function call
```
[the remark this answers is not in the document]
Comment (ndmckinley) [reply provider/ansible/request.rb]: Eh, I already tried removing the complexity there and it works fine.  :)  Pushing that up now.

**Comment by rambleraptor** [comment]
Don't worry too much about making the formatting perfect. I'm planning on doing a whole bunch of refactoring now that we have a good code formatter.

I'm super excited.

**Comment by ndmckinley** [comment]
Awesome, I'll submit when this is done, then.  :)

**Pull request merged**